test/server/util/twilio_mocks.js

```diff
@@ -35,15 +36,20 @@ export function returnUpdate(callSid, updateType) {
     to: '+14086428264',
     toFormatted: '(408) 642-8264',
     uri: `/2010-04-01/Accounts/${TWILIO_ACCOUNT_SID}/Calls/${callSid}.json`
-  }
+  };
   switch (updateType) {
     case CALL_COMPLETE:
       clientUpdateObject.duration = '14';
-      clientUpdateObject.endTime = '2017-09-11T18:20:44.000Z'
-      clientUpdateObject.status = 'completed'
+      clientUpdateObject.endTime = '2017-09-11T18:20:44.000Z';
+      clientUpdateObject.status = 'completed';
       return clientUpdateObject;
     case CONNECT_CONTACT_URL:
-      clientUpdateObject.status = 'in-progress'
+      clientUpdateObject.status = 'in-progress';
+      return clientUpdateObject;
+    case DISCNNECT_CONTACT_URL:
+      clientUpdateObject.status = 'completed';
```

Comment:
So yeah we don't actually terminate the volunteer call. So we'll have to modify this test too. There really won't be much that changes in the call object IIRC since we're transparently changing the URL which isn't reflected in the response from the Twilio API or client.

Reply:
hmm ok, I might have a few questions about this in a bit